common/oatbox/validator/ExtendedValidatorInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA;
+ *
+ */
+namespace oat\oatbox\validator;
+
+interface ExtendedValidatorInterface extends ValidatorInterface
+{
+    /**
+     * @param array $values
+     * @param string $currentPropertyUri
+     */
+    public function populateAdditionValues(array $values, $currentPropertyUri);
```

Comment:
I would change the interface to
`public function populateWithFormValues(array $values, $currentPropertyUri);`
But in general I think the solution should be reviewed by an architect, to decide if this is the proper way to pass the modified resource id to a validator.

Reply:
totally agree, we don't need this interface

Reply:
But if we need to validate some more complex form. For example with name1 and name2 and a least one field should be filled. In this case validation should be aware of other form values